packages/core/src/sync-realtime/index.ts

```diff
@@ -624,7 +628,7 @@ export const createRealtimeSync = (
 
             // New block is exactly one block ahead of the local chain.
             // Attempt to ingest it.
-            await handleBlock({ block, ...rest });
+            handleBlock({ block, ...rest });
```

Comment:
Happy path worker for the realtime queue (bottleneck for handling a new block) is now synchronous